### ISSUE TITLE
Eye icon on Usage button on Progress and Ranking page should be changed

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1193]
The eye icon is already used in the dashboard for the project preview buttons. As per accessibility standards, icons should not be re-used for other purposes within the application. Change the eye icon on the Usage button of the Progress and Ranking view to something else.

**Comment by rmmayo** [comment]
PR: #1435

**Comment by mosefer** [comment]
:+1:
